Automattic/wp-calypso
# Signup: Add reskinSignupFlow a/b test

#### Changes proposed in this Pull Request

This PR impldements an A/B test to show a different look of the `onboarding` signup flow to the variation group. See pau2Xa-1x4-p2 for more details.

cc @aneeshd16 

#### Testing instructions



* Go to `/start/new`.
* Make sure you're assigned to `reskinned` group of `reskinSignupFlow` a/b test.
  * Note: this test is intended to work only for non-EN and non-ES users. (p1596055144383000-slack-marketing-health).
* The signup flow should look as below.

<img width="866" alt="new-signup" src="https://user-images.githubusercontent.com/212034/87685294-2e234b00-c7be-11ea-9893-2779d65a45ed.png">
<img width="866" alt="new-domains" src="https://user-images.githubusercontent.com/212034/87685302-324f6880-c7be-11ea-9e4a-d3179f673544.png">
<img width="866" alt="new-domain-selection" src="https://user-images.githubusercontent.com/212034/87685310-34b1c280-c7be-11ea-9767-13d1c9261e7d.png">
<img width="866" alt="new-plans" src="https://user-images.githubusercontent.com/212034/87685317-37acb300-c7be-11ea-86bf-dc1d6116029f.png">
<img width="866" alt="new-processing" src="https://user-images.githubusercontent.com/212034/87685325-39767680-c7be-11ea-9f00-7178855b2a80.png">
